srcpkgs/fah/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'fah'
+pkgname=fah
+version=7.5.1
+revision=1
+archs="x86_64"
+hostmakedepends="python"
+depends="python desktop-file-utils hicolor-icon-theme"
+short_desc="Folding@home distributed computation power for research"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="custom:Proprietary"
+homepage="https://foldingathome.org"
+distfiles="https://download.foldingathome.org/releases/public/release/fahclient/debian-stable-64bit/v${version%.*}/fahclient_${version}_amd64.deb
+ https://download.foldingathome.org/releases/public/release/fahcontrol/debian-stable-64bit/v${version%.*}/fahcontrol_${version}-1_all.deb
+ https://download.foldingathome.org/releases/public/release/fahviewer/debian-stable-64bit/v${version%.*}/fahviewer_${version}_amd64.deb"
+checksum="efe47061d328667259c1815b98bc6c37da339363fb8e9386673f6a96f4d37d26
+ c9d37823abe666923d903fbf4d7c6059cf8042a208954bc0044c9c5a6635b4d8
+ c87d331a35fc68d04427996aef05b275ca69dab3bef4087efaaf5e27e95d7b29"
+nostrip=yes
+repository="nonfree"
+restricted=yes
+system_accounts="_fah"
+_fah_homedir="/usr/libexec/_fah"
+_fah_descr="Folding@Home user"
+
+do_extract() {
+	ar x ${XBPS_SRCDISTDIR}/${pkgname}-${version}/fahclient_${version}_amd64.deb
+	bsdtar xf data.tar.xz
```

Comment:
i suggest to pipe data.tar.xz to tar instead of saving it to disc.
`ar p foo.deb data.tar.xz | tar -x -f -`

Reply:
Done.